packages/base/any/kernels/4.9-lts/configs/x86_64-all/x86_64-all.config

```diff
@@ -1221,7 +1221,7 @@ CONFIG_VIRTIO_BLK=y
 #
 # EEPROM support
 #
-CONFIG_EEPROM_AT24=m
+CONFIG_EEPROM_AT24=y
```

Comment:
This config must remain as a module as it is loaded by other platforms with different module params. 
Please keep as a module and load it if necessary in your platform class.

Reply:
the changes was made in the commit